apps/admin-ui/src/i18n/messages.ts

```diff
@@ -816,6 +816,12 @@ const translations: ITranslations = {
         "reservationBeginsDate must be before end": [
           "Varausajan tulee alkaa ennen kuin se päättyy.",
         ],
+        "duration can't be less than reservation start interval": [
+          "Kesto ei voi olla pienempi kuin varauksen alkamisväli.",
+        ],
+        "duration must be a multiple of the reservation start interval": [
+          "Kesto on oltava varauksen alkamisvälin moninkertainen.",
```

Comment:
This looks to me like incorrect Finnish - I think the proper translation would be "Keston on oltava varauksen alkamisvälin kerrannainen".